test/plugin/base_test.rb

```diff
@@ -2199,8 +2318,10 @@ def verify_json_log_entries(n, params, payload_type = 'textPayload')
             log_name
         end
         assert_equal params[:resource][:type], resource['type']
-        check_labels resource['labels'], params[:resource][:labels]
-        check_labels labels, params[:labels]
+
```

Comment:
I'd put this blank line between the two `check_labels` calls instead, to group the resource checks together...

Reply:
Done